Kernel/Language/es_MX.pm

```diff
@@ -6389,7 +6389,7 @@ Thanks for your help!
 ' => '
 Estimado cliente,
 
-Lamentablemente no hemos podido detectar un número de ticket válido 
```

Comment:
Yes, finally eliminating that trailing space. Let's see whether WebLate adds it again.